pkg/build/strategies/sti/sti.go

```diff
@@ -136,7 +136,7 @@ func (b *STI) Build(config *api.Config) (*api.Result, error) {
 		}
 	}
 
-	glog.V(1).Infof("Building %s", config.Tag)
+	glog.V(1).Infof("Running STI script in the builder image")
```

Comment:
can you please use the `config.Tag` instead of "the builder image" or just add that information?